relayer/src/chain/mock.rs

```diff
@@ -61,6 +61,14 @@ pub struct MockChain {
     event_receiver: EventReceiver,
 }
 
+impl MockChain {
+    fn trusting_period(&self) -> Duration {
+        self.config
+            .trusting_period
+            .unwrap_or(Duration::from_secs(14 * 24 * 60 * 60)) // 14 days
```

Comment:
To make Clippy happy :)

```suggestion
            .unwrap_or_else(|| Duration::from_secs(14 * 24 * 60 * 60)) // 14 days
```